### PR TITLE
Fix OLAP stats

### PR DESCRIPTION
**Pull request by k-morozov**

### CODE REVIEW

**Comment by k-morozov** [comment]
Im going to add some utests (for many tables in table store).

**Comment by k-morozov** [comment]
There is a problem related with a large count tables in tablesstore. Some tables doesn't init by stats.

File: ydb/core/protos/tx_datashard.proto [hunk 1]
```diff
@@ -806,6 +806,8 @@ message TEvPeriodicTableStats {
     optional uint64 TableOwnerId = 12;
 
     optional bool IsDstSplit = 13;
+
+    repeated TEvPeriodicTableStats ColumnTables = 14;
```
Comment (nsofya) [inline ydb/core/protos/tx_datashard.proto:810]: Давай без Column. Статистика же общего вида

Comment (k-morozov) [reply ydb/core/protos/tx_datashard.proto]: ок

**Review by ivanmorozov333 — APPROVED** [review]

**Comment by ijon** [comment]
I don't understand, why this pr, which touches core mechanics of stats collection in schemeshard for all table types and not only for olap tables: 1) named highly inaccurate "fix olap", and 2) had been merged without approve from me while I explicitly registered my interest in reviewing it.

**Pull request merged**